uwblueprint/sistering
# Implement Organization-Wide Calendar for Admins & Employees

## Ticket link

Closes #535 
Closes #555 



## Implementation description
* Added a new query `shiftsWithSignupsAndVolunteers` (of the shiftService) to return all posting's shifts with signups and volunteers.
* Created the org wide calendar page at `/admin/calendar`. This calendar displays the **scheduled/past** signups and volunteers from all postings across Sistering.
* Some clean-up work
  * Renamed IShiftService.ts to shiftService.ts on the backend to keep interface file naming convention.
  * Re-ordered routes for better organization
* Extended `getUserBranchesByUserId` functionality to also work for employees and admins (this is used in `getPostings`, which is then called in the frontend for employees and admins)



## Steps to test
1. Navigate to [](http://localhost:3000/admin/calendar) as a super admin, should see all scheduled and past postings in any branch.
2. Navigate to the same screen as an admin, should only see scheduled and past postings from your own branch.



## What should reviewers focus on?
* Correctly displaying events


## Checklist
